webui/react/src/pages/JobQueue/JobQueue.table.tsx

```diff
@@ -134,8 +137,13 @@ export const columns: ColumnDef<Job>[] = [
     dataIndex: 'submissionTime',
     defaultWidth: DEFAULT_COLUMN_WIDTHS['submissionTime'],
     key: 'submitted',
-    render: (_: unknown, record: Job): ReactNode =>
-      record.submissionTime && relativeTimeRenderer(record.submissionTime),
+    render: (_: unknown, record: Job): ReactNode => {
+      if ('submissionTime' in record) {
+        return record.submissionTime && relativeTimeRenderer(record.submissionTime);
+      } else {
+        return OMITTED_STR;
+      }
+    },
```

Comment:
**question suggestion:** this and the user version can maybe be abstracted into something like `omitRenderer` and put in `components/Table/Table`? Maybe not worth the effort if it requires a lot of TS wrangling

Reply:
good idea. maybe there is a better way that'll let the final renderer know about the resolved type but this works:
 https://github.com/determined-ai/determined/pull/7001/commits/31fd5a081f0a2d43077a1fbf5a7907621f9061ae
https://github.com/determined-ai/determined/pull/7001/commits/27e4984ea6a26cfd0c18057b624b37c5e9e9ea51
 2 files changed, 14 insertions(+), 23 deletions(-)